.github/workflows/post-dependabot.yml

```diff
@@ -41,3 +41,25 @@ jobs:
           git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}
           git commit -m "Update NOTICE.txt"
           git push
+
+  manage-pr-reviewers:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add reviewers (obs-infraobs-integrations) if PR has Team:Obs-InfraObs label
```

Comment:
Will this work if no checkout? I think you need to checkout the source code so the `gh` cli expects a workspace to be there, unless you use `-R`.

Reply:
Yeah, I think so. I checked some similar changes and they had no checkout step.

https://github.com/up-for-grabs/up-for-grabs.net/blob/gh-pages/.github/workflows/review-pull-request-state.yml
https://github.com/carbon-language/carbon-lang/blob/trunk/.github/workflows/proposal_labeled.yaml